src/js/components/Settings/VoterEmailAddressEntry.jsx

```diff
@@ -602,20 +602,24 @@ class VoterEmailAddressEntry extends Component {
               {voterEmailAddressFromList.email_ownership_is_verified ?
                 null : (
                   <SecondRowPhoneOrEmail>
-                    <span
-                      className="u-link-color u-cursor--pointer u-no-break"
-                      onClick={() => this.reSendSignInCodeEmail(voterEmailAddressFromList.normalized_email_address)}
-                    >
-                      Send verification again
-                    </span>
-                    {allowRemoveEmail && (
-                      <TrashCan
-                        className="u-link-color u-cursor--pointer"
-                        onClick={() => this.removeVoterEmailAddress(voterEmailAddressFromList.email_we_vote_id)}
+                    <SecondRowPhoneOrEmailDiv key={`${voterEmailAddressFromList.email_we_vote_id}-internal`}>
```

Comment:
I think this key may need to be moved to the `SecondRowPhoneOrEmail` div on the line above.